changelogs/fragments/1632-Validate_to_restore_keep_orginial_hlq.yml

```diff
@@ -0,0 +1,3 @@
+trivial:
```

Comment:
Since you're changing a little bit what one of the options can do, I would use `minor_changes` instead of `trivial`